mingus/midi/midi_track.py

```diff
@@ -256,7 +257,7 @@ def set_key(self, key="C"):
 
     def key_signature_event(self, key="C"):
         """Return the bytes for a key signature event."""
-        if key.islower():
+        if str(key).islower():
```

Comment:
Nice. I think this solves it for now. In the future we should use stricter types.